src/Mod/Material/Templatematerial.yml

```diff
@@ -265,7 +270,7 @@
       Type: 'String'
       URL: ''
       Description: "Vendor of the material"
-- UserDefined:
+- User defined:
```

Comment:
Is this really safe? In any case it most certainly would need further changes. I suggest you revert this line, and make another PR with it if applicable